addons/mrp/models/mrp_production.py

```diff
@@ -1690,7 +1687,7 @@ def _get_backorder_mo_vals(self):
             'orderpoint_id': self.orderpoint_id.id,
         }
 
-    def _split_productions(self, amounts=False, cancel_remaining_qty=False, set_consumed_qty=False):
+    def _split_productions(self, amounts=False, cancel_remaining_qty=False, set_consumed_qty=False, sml_create=False):
```

Comment:
set_consumed_qty should not create the stock.move.line for components. Why do you need a new arg